homeassistant/components/tankerkoenig/const.py

```diff
@@ -10,3 +10,12 @@
 DEFAULT_SCAN_INTERVAL = 30
 
 FUEL_TYPES = {"e5": "Super", "e10": "Super E10", "diesel": "Diesel"}
+
+ATTR_BRAND = "brand"
+ATTR_CITY = "city"
+ATTR_FUEL_TYPE = "fuel_type"
+ATTR_HOUSE_NUMBER = "house_number"
+ATTR_POSTCODE = "postcode"
+ATTR_STATION_NAME = "station_name"
+ATTR_STREET = "street"
+ATTRIBUTION = "Data provided by https://creativecommons.tankerkoenig.de"
```

Comment:
Why have attribution link to creative commons URL and not their website ? Seems main website is more useful ?

Reply:
this was just moved from `sensor.py` to `const.py` , but i agree, the main website should be placed here